README.md

```diff
@@ -1,84 +1,126 @@
+| configuration.json                       | Default _configuration.json_ |
+| fieldMapping.json                        | Default _fieldMapping.json_ |
+| assets/YsisMapping.csv                   | Example Ysis discipline _mapping csv_ |
+| assets/ConditionalNotification.mjml      | Example Discipline has changed _notification_ |
 
->:exclamation:It is not possible to change the discipline of an existing account. Therefore, during the `update` life-cycle a change in discipline will launch a conditional event which sends an email to the Ysis administrator.
+## Getting Started
 
-- In Ysis each account has a discipline that acts as the account type.
-- When a person requires a different (or an extra discipline), a new user account must be created with the new discipline. Manual actions by the Ysis administrator are needed.
+### Prerequisites
 
-## Introduction
-The interface to communicate with Profit is through a set of GetConnectors, which is component that allows the creation of custom views on the Profit data. GetConnectors are based on a pre-defined 'data collection', which is an existing view based on the data inside the Profit database. 
+- A server with a local agent is required.
+- The outgoing IP address of the HelloID agent server must be whitelisted by GeriMedica.
+- A mapping between function and discipline is created.
+- The end date for active accounts should be cleared (see [End date must be cleared](#end-date-must-be-cleared)
 
-For this connector we have created a default set, which can be imported directly into the AFAS Profit environment.
-The HelloID connector consists of the template scripts shown in the following table.
+> [!TIP]
+> You can validate the outgoing IP address on the HelloID agent server with the following PowerShell script:
+> ```powershell
+> $ip = Invoke-RestMethod -uri "https://ipinfo.io/json" -method get
+> Write-Verbose -Verbose "$($ip.ip)"
+> ```
 
-| Action                          | Action(s) Performed   | Comment   | 
-| ------------------------------- | --------------------- | --------- |
-| create.ps1                      | Create or correlate Ysis account  | Create or correlates an Ysis account. If correlated and UpdateOnCorrelate is configured, the update will be processed |
-| enable.ps1                      | Activate Ysis account  | Activates Ysis account |
-| update.ps1                      | Update Ysis account  | Update on Ysis account. Conditional event on discipline change. |
-| disable.ps1                     | Deactivate Ysis account  | Deactivates Ysis account |
-| delete.ps1                      | Archive Ysis account  | Archives the Ysis account |
+### Correlation configuration
 
-<!-- GETTING STARTED -->
-## Getting Started
+The correlation configuration is used to specify which properties will be used to match an existing account within _HelloID-Conn-Prov-Target-Ysis to a person in _HelloID_.
```

Comment:
Not sure if this is still the case. Last time only correlating on EmployeeNumber was possible. Maybe add this to the readme?

Reply:
Added